### ISSUE TITLE
Failure to use attendance_category

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/universal-xapi-translator#16]
From @arc12 

We know that D&G record a letter code against each attendance. Andy and Duncan talked us through it. Not found in statements

**Comment by michaelwebbjisc** [comment]
@arc12 can you expand on this?  Do you mean there was extra data which was could be added in 

"http://xapi.jisc.ac.uk/attendance_category 

ie

 "result":{
        "completion":true,
		
	 "extensions":{
		  "http://xapi.jisc.ac.uk/attendance_late":1,
		  "http://xapi.jisc.ac.uk/attendance_category":"1",
		 }
    }

**Comment by arc12** [comment]
Yes, things looking like standard school register codes: "O, X, L, E, C"

Covered situations like absent but notified in advance, left early, authorised absence.

It was pointed out that there is a grey area which a binary view of attendance didn't capture. Late flag gets some of this, but not all

**Comment by michaelwebbjisc** [comment]
Ok, have contacted D&G to get the extra data.
